benchmark/events/ee-emit-mixed.js

```diff
@@ -0,0 +1,23 @@
+var common = require('../common.js');
+var EventEmitter = require('events').EventEmitter;
+
+var bench = common.createBenchmark(main, {n: [2e6]});
+
+function main(conf) {
+  var n = conf.n | 0;
+
+  var ee = new EventEmitter();
+
+  for (var k = 0; k < 10; k += 1) {
+    if (k % 2)
+      ee.once('dummy', function() {});
+    else
+      ee.on('dummy', function() {});
+  }
+
+  bench.start();
+  for (var i = 0; i < n; i += 1) {
+    ee.emit('dummy');
```

Comment:
just throwing ideas at the wall, but i'm thinking some sort of `Object.freeze()` (maybe [deep-freeze](https://github.com/substack/deep-freeze)) on the event emitter, which would (in non-strict mode) i believe silently fail when it is changed, leaving us the same object in each loop - thats the only idea that comes to mind that would *only* benchmark the `.emit()`, what do you think?